### PR TITLE
Register Subdomain: halloween

### PR DESCRIPTION
**Pull request by ghost**
Added the domain: halloween

### CODE REVIEW

File: domains/time.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+    {
+      "owner": {
+        "username": "eastisup",
+        "email": "hi@eastisup.com"
+      },
+
+      "record": {
+        "CNAME": "e1c0a366-dc65-4e70-9219-2f7213a8803d.id.repl.co",
+        "TXT": "replit-verify=e1c0a366-dc65-4e70-9219-2f7213a8803d"
```
Comment (WilliamDavidHarrison) [inline domains/time.json:10]: Sorry we don't support TXT verification

File: domains/halloween.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "halloweenphile",
+        "email": "weeksb@bentonvillek12.org"
+      },
+      "record": {
+        "CNAME": "https://halloweenphile.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/halloween.json:8]:
```suggestion
        "CNAME": "halloweenphile.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/time.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+    {
+      "owner": {
+        "username": "eastisup",
+        "email": "hi@eastisup.com"
+      },
+
+      "record": {
+        "CNAME": "e1c0a366-dc65-4e70-9219-2f7213a8803d.id.repl.co",
```
Comment (WilliamDavidHarrison) [inline domains/time.json:9]:
```suggestion
        "CNAME": "e1c0a366-dc65-4e70-9219-2f7213a8803d.id.repl.co"
```

File: domains/time.json (hunk 1 above)
Comment (WilliamDavidHarrison) [inline domains/time.json:10]:
```suggestion
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
Hello world website, no response, uses TXT verification.

If you want a domain please finish your website, review our guidelines and make a new PR.

**Pull request closed without merging**